### ISSUE TITLE
NTR: Apical ground-glass opacification

### ISSUE THREAD

**Issue by LCCarmody** [obophenotype/human-phenotype-ontology#6251]
**Preferred term label:**
Apical ground-glass opacification

**Synonyms**

**Definition (free text, please give PubMed ID)**
'The typical feature was multiple lobular ground-glass opacity and subsegmental areas of consolidation in COVID-19 patients. CT scan may be a reliable test for screening COVID-19 cases.'
PMID:32161968
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Your nano-attribution (ORCID)**

**Comment by rajamazumder** [comment]
This will be great. I would be curious to know how many of the ~150 terms for potential COVID-19 biomarkers would also be considered as HPO terms. Any help in that area is appreciated. The website https://data.oncomx.org/covid19 is very basic and work in progress. But you should be able to download the table.

**Comment by drseb** [comment]
@LCCarmody  as subclass of https://hpo.jax.org/app/browse/term/HP:0025179?

**Comment by pnrobinson** [comment]
This is not a good definition. Let's go with 
def: Increased attenuation of the top (apex) of the lung on computed tomography (CT) with preserved bronchial and vascular markings.

Before we make the new term we need to see if the definition should be restricted to HR CT or not, I think we may want to revise the labels of some of the previous high resolution CT terms as well.

**Comment by mgriese** [comment]
GGO is a radiological term, typically used with chest CT imaging. However it can also be applied for chest x-rays. I would not include the localisation of GGO in the term (i.e. apical), as it can be anywhere.

A hazy increase in lung opacity on chest imaging that is not associated with obscuration of underlying
vessels and can therefore be differentiated from air-space consolidation. It can reflect the presence of minimal interstitial thickening, partial air-space filling, partial collapse of alveoli or increased capillary blood volume. Ground-glass opacity can be diffuse, patchy, or nodular. 
PMID: 6380245, 8685321

**Comment by pnrobinson** [comment]
Leave off "apical"
HCRT as child

**Comment by pnrobinson** [comment]
We will take care if this in https://github.com/obophenotype/human-phenotype-ontology/issues/6690
